src/Native/AnyOS/brotli/tools/brotli.c

```diff
@@ -943,10 +943,9 @@ static BROTLI_BOOL CompressFiles(Context* context) {
       uint32_t lgwin = DEFAULT_LGWIN;
       /* Use file size to limit lgwin. */
       if (context->input_file_length >= 0) {
-        int32_t size = 1 << BROTLI_MIN_WINDOW_BITS;
         lgwin = BROTLI_MIN_WINDOW_BITS;
-        while (size < context->input_file_length) {
-          size <<= 1;
+        while (BROTLI_MAX_BACKWARD_LIMIT(lgwin) <
+               (uint64_t)context->input_file_length) {
```

Comment:
This 1k LOC file is a CLI tool and is not used in CoreFX - maybe it should be deleted?

Reply:
I don't understand this comment. Can you please elaborate and provide more context?

Reply:
@ahsonkhan `brotli.c` is the only file we don't need from `google\brotli` so I suggest to delete it and ignore it in further brotli updates. the file is 34kb. (btw we use brotli files from corefx repo in mono)

Reply:
@ahsonkhan brotli.c is the source for the brotli.exe cmdline util, which we don't need at all

Reply:
This was previously discussed in https://github.com/dotnet/corefx/pull/30295

As pointed out there, we don't need anything in the `tools` nor `fuzz` folders here.  Previous decision was to keep them, though.